packages/typedoc-plugin-appium/lib/output/theme/utils.ts

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import fs from 'node:fs';
+import path from 'node:path';
+import Handlebars from 'handlebars';
+import {ContainerReflection, PageEvent, ReflectionKind} from 'typedoc';
+import {AppiumPluginReflectionKind} from '../../model';
+
+const RESOURCES_PATH = path.join(__dirname, '..', '..', '..', 'resources');
+const TEMPLATE_PATH = path.join(RESOURCES_PATH, 'templates');
+const PARTIALS_PATH = path.join(RESOURCES_PATH, 'partials');
+
+const Partials = {
+  command: 'command.hbs',
+  executeCommand: 'execute-command.hbs',
+} as const;
+
+function registerPartials() {
+  for (const [name, filename] of Object.entries(Partials)) {
+    Handlebars.registerPartial(name, fs.readFileSync(path.join(PARTIALS_PATH, filename), 'utf8'));
+  }
+}
+
+registerPartials();
+
+export enum Template {
+  Commands = 'commands.hbs',
+}
+
+export const compileTemplate = _.memoize((template: Template) => {
+  const templatePath = path.join(TEMPLATE_PATH, template);
+  return Handlebars.compile(fs.readFileSync(templatePath, 'utf8'));
```

Comment:
cannot we use async calls?

Reply:
No... `compileTemplate` is called by `AppiumTheme#getTemplate`, which is called by `AppiumTheme#mappings` which is a computed property.  It can't return a `Promise` because the parent class expects it to be a value.